frontend/device/pocketbook/device.lua

```diff
@@ -103,6 +109,17 @@ function PocketBook:initNetworkManager(NetworkMgr)
     end
 end
 
+function PocketBook:getSoftwareVersion()
+    version =  ffi.string(inkview.GetSoftwareVersion())
```

Comment:
`local version`

Otherwise you create a variable in the global namespace, which besides littering it is also bad for garbage collection.